README.md

```diff
@@ -166,6 +166,33 @@ $money = Money::fromDecimal(40.25, USD::class);
 $money->formatted(['decimalSeparator' => ',', 'prefix' => '$ ', 'suffix' => ' USD']);
 ```
 
+There is also a `->rawFormatted()` if you wish to use [math decimals](#math-decimals) instead of [display decimals](#display-decimals).
```

Comment:
```suggestion
There's also `->rawFormatted()` if you wish to use [math decimals](#math-decimals) instead of [display decimals](#display-decimals).
```